hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -758,10 +759,12 @@ public void testListBucket() throws Exception {
     final String hostPrefix = OZONE_OFS_URI_SCHEME + "://" + omServiceId;
     OzoneConfiguration clientConf =
             getClientConfForOFS(hostPrefix, cluster.getConf());
+    int pageSize = 20;
+    clientConf.setInt(OZONE_FS_LISTING_PAGE_SIZE, pageSize);
     OzoneFsShell shell = new OzoneFsShell(clientConf);
 
     String volName = "testlistbucket";
-    int numBuckets = 1025;
+    int numBuckets = pageSize;
```

Comment:
Should it be `pagesize + 1`?

Reply:
`pageSize` will also trigger the bug.

Reply:
@adoroszlai I wonder if there is a timeout wrapper on some code block in unit test, so that it can fail fast?

Reply:
Add `@Timeout(seconds)` on the test method, e.g.

https://github.com/apache/ozone/blob/97038ef0973f86aacaa5d3c3c1e3a22febd3b947/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/freon/TestRandomKeyGenerator.java#L72-L73

Reply:
@adoroszlai Thanks for the advice. Updated, PTAL.

I was also wondering if there is something like 

`timout.wrapper(ToolRunner.run(shell, new String[]{"-ls", "/" + volName}));`